### PR TITLE
Stats over 3 digits overlap (fixes #10033)

### LINKED ISSUE #10033: Stats over 3 digits overlap
**From the Bug Build**

On the stats tab, in the stats section at the bottom, my intelligence is 5 characters long (105.5) and overlaps with the info on the right of the block in which it's contained, partially obscuring the other info.
See https://i.imgur.com/OmbbgcR.png

s

### PR DESCRIPTION
**Pull request by Mahendran-Nadesan**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #10033

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Attribute stats in profile are now rounded to whole numbers.

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: 92d94cfe-3623-4242-b07b-73f02f88fde1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@ newrelic_agent.log
 Vagrantfile
 TODO
 *.log
+.gitattributes
```
Comment (paglias) [inline .gitignore]: why did you add this to the gitignore?

File: website/client/filters/floorWholeNumber.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function floorWholeNumber (val) {
+    return Math.floor(val);
+  }
```
Comment (paglias) [inline website/client/filters/floorWholeNumber.js]: the parenthesis should not be indented

**Review by paglias — CHANGES_REQUESTED** [review]
left two small comments then this should be ready to go, thanks!

File: .gitignore (hunk 1 above)
Comment (Mahendran-Nadesan) [reply .gitignore]: For local testing, then forgot to remove it. Will do.

**Comment by paglias** [comment]
@Mahendran-Nadesan  thanks for this! I'm merging the PR and awarding your contributor tier.

Note that further tiers require increasing amounts of work from one to the next, so Tier 2 may require a couple of PRs or a larger PR to attain. But keep helping out and we'll express our gratitude accordingly!

**Comment by Mahendran-Nadesan** [comment]
Much appreciated. Was a pleasure to finally contribute to something!

**Comment by Mahendran-Nadesan** [comment]
Quick query (out of curiosity) - when do the changes in develop get merged with the release?

**Comment by paglias** [comment]
@Mahendran-Nadesan if everything goes as planned we tend to merge PRs on Friday or the weekend and deploy them on Thursday
